src/App.jsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import './App.css'
+import play from './access/paly.svg'
+import pause from './access/pause.svg'
+import resetTime from './access/time.svg'
+import arrow from './access/arrow.svg'
+import {connect} from "react-redux";
+import {addBreakLength, addSessionLength, reset, startStopTimer} from "./redux/appReduxer";
+import {getFormate} from "./utils";
+
+let Setting = ({title, value, upWork, upRest, downWork, downRest, category}) => {
```

Comment:
Don't use let statement if you're not gonna change this. It makes sense it is going to be changed within app and thus can confuse your teammate

Reply:
Changed